.github/workflows/scripts/inspect_remote_img.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+inspect_remote_img() {
+  GH_TOKEN=$1
+  IMAGE=$2
+
+  TAG=$(echo $IMAGE | rev | cut -d: -f1 | rev)
+  IMAGE_REPO=$(echo $IMAGE | rev | cut -d: -f2- | rev)
+
+  PACKAGE=$(echo $IMAGE_REPO | rev | cut -d/ -f1 | rev)
+  ORG=$(echo $IMAGE_REPO | rev | cut -d/ -f2 | rev)
+
+  top_manifest_digest=$(curl -s -H "Authorization: Bearer $(echo $GH_TOKEN | base64)" "https://ghcr.io/v2/$ORG/$PACKAGE/manifests/$TAG" | jq -r .manifests[0].digest)
```

Comment:
This might not work in the future when we start to build multi-arch containers for PAX and T5X, since `manifests[]` will contain at least two manifests corresponding to the two architectures, respectively.

Reply:
Gotcha. Do you have an example of an image with mutliple manifests I can use as a reference to update this code to select the right manifest?

Or are you suggesting we just merge in and fix later (create GH issue)

Reply:
Issue created: #263.